### PR TITLE
Clarify that the tool runs offline

### PR DESCRIPTION
**Pull request by ievans**
This was an extremely common objection to Bento, and I don't think we make it clear enough in the current iteration of the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,14 +41,14 @@
   </a>
 </p>
 
-Semgrep is a simple and flexible [command-line](#installation) and [online](https://semgrep.live/) tool for static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase.
+Semgrep is a [command-line](#installation) tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase. You can try it now with our [interactive live editor](https://semgrep.live/).
```
Comment (pabloest) [inline README.md]: I agree with specifying that this is an offline thing.
```suggestion
Semgrep is a [command-line](#installation) tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase. You can try it now with our [interactive live editor](https://semgrep.live/editor).
```

File: README.md [hunk 2]
```diff
@@ -41,14 +41,14 @@
...
 
-Semgrep combines the convenient and iterative style of `grep` with the powerful features of an Abstract Syntax Tree (AST) matcher. Easily find function calls, class or method definitions, and more without having to understand ASTs or wrestle with regexes.
+Semgrep combines the convenient and iterative style of `grep` with the powerful features of an Abstract Syntax Tree (AST) matcher and limited dataflow. Easily find function calls, class or method definitions, and more without having to understand ASTs or wrestle with regexes.
 
 Visit [Installation](#installation) and [Usage](#usage) to get started.
 
 ## Installation
-> Want to skip installation? You can run Semgrep online via [semgrep.live](https://semgrep.live/).
+> Want to skip installation? You can run Semgrep via a live editor at [semgrep.live](https://semgrep.live/).
```
Comment (pabloest) [inline README.md]:
```suggestion
> Want to skip installation? You can run Semgrep via our interactive live editor at [semgrep.live](https://semgrep.live/editor).
```
I like the use the word interactive above so matching it here, too.

File: README.md (hunk 1 above)
Comment (ievans) [reply README.md]: thanks @pabloest , good call

**Comment by brendongo** [comment]
Should we clarify it does use network to download a config if you're using an online configuration?

**Comment by ievans** [comment]
Maybe? I think if the config is https://foo that makes it pretty obvious, but --config=r2c might be a bit more subtle. Maybe when we introduce --config=r2c we can say (this will load the default config from r2c over the network, but it doesn't send your code anywhere). Or we could say "analysis runs offline"

**Comment by dlukeomalley** [comment]
I agree with the spirit of this and will comment more tomorrow morning. I liked the idea of placing people on the semgrep.live home page vs. piping them straight to the editor, it offers a broader intro the registry + editor.

**Comment by brendongo** [comment]
If you're talking about an interactive editor it doesnt make sense to send people to a page that isn't the interactive editor

**Comment by dlukeomalley** [comment]
Hey @brendongo, it sounds like the original intent of my paragraph and comment may have been misinterpreted. I was not initially thinking of talking about the interactive editor, but instead exposing people to Semgrep's Registry AND Live Editor. I thought both are interesting and constitute the online experience.

Are there specific questions you have for me so I can clarify for you what I had in mind? Would love to improve these paragraph so it's clearer for readers like yourself.

File: README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ Semgrep exists because:
 2. The future of security involves automatically guiding developers towards a “paved road” made of default-safe frameworks (i.e. [React](https://reactjs.org/) or Object-relational Mappers)
 3. `grep` isn’t expressive enough and traditional static analysis tools (SAST) are too complicated/slow for paved road automation
 
-The AppSec, Developer, and DevOps community deserves a static analysis tool that is fast, easy to use, code-aware, multi-lingual, and open source!
+The AppSec, Developer, and DevOps communities deserves a static analysis tool that is fast, easy to use, code-aware, multi-lingual, and open source!
```
Comment (rcoh) [inline README.md]: typo: `deserve`
```suggestion
The AppSec, Developer, and DevOps communities deserve a static analysis tool that is fast, easy to use, code-aware, multi-lingual, and open source!
```

**Comment by brendongo** [comment]
@dlukeomalley the text around the link has the intention of getting the user to try out semgrep and in that case I don't think we want to link the user to a landing page. It just seems weird to go "try it here" and then they can't actually try out the tool on the page they were sent to

**Comment by brendongo** [comment]
I think for now just going to get this PR in since the text around these links used to assume that semgrep.live went to the live editor and it makes sense to point them to semgrep.live/editor now that the location of the live editor has shifted. We can discuss re: changing the intention of the text to point to a wholistic online experience in a follow up PR

**Review by brendongo — APPROVED** [review]

**Pull request merged**